File: chapter02/manyconditionswithswitch.php
<?php
$letter = $argv[1] ?? 'x';

switch ($letter){
	case 'a':
		echo 'archive'; break;
	case 'b':
		echo 'brief'; break;
	case 'c':
		echo 'create'; break;
	case 'd':
		echo 'delete'; break;
	case 'e':
		echo 'extract'; break;
	case 'f':
		echo 'format'; break;
	default:
		echo 'command not found';
}
echo "\n";	

File: chapter02/manyconditionswithswitch.c
#include <stdio.h>

void main(int argc, char *argv[])
{
	char letter = 'x';
	
	if (argc > 1) {
		letter = *argv[1];
	}

	switch (letter){
		case 'a':
			printf("archive"); break;
		case 'b':
			printf("brief"); break;
		case 'c':
			printf("create"); break;
		case 'd':
			printf("delete"); break;
		case 'e':
			printf("extract"); break;
		case 'f':
			printf("format"); break;
		default:
			printf("command not found");
	}
	printf("\n");	
}
